lib/log.js

```diff
@@ -26,9 +28,16 @@ module.exports = function log(...msg) {
   console.log(d, ...msg);
 };
 
+const throttledConsoleError = throttle(console.error, 10000, { trailing: false });
```

Comment:
Errors returned by `captureException` will be throttled and logged one per 10 seconds. Would you like to change this time or be able to configure it by an environment variable? Btw, this is a really nice article about debouncing and throttling :-) https://css-tricks.com/debouncing-throttling-explained-examples/